Client/src/app/modules/posts/components/posts-create-form/posts-create-form.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+import { PostsApiService } from '../../services/posts.api.service';
+
+@Component({
+  selector: 'app-posts-create-form',
+  templateUrl: './posts-create-form.component.html',
+  styleUrls: ['./posts-create-form.component.scss']
+})
+export class PostsCreateFormComponent {
+  postCreationForm = new FormGroup({
+    title: new FormControl(''),
+    content: new FormControl('')
+  });
```

Comment:
Can a user create a post with no title and content?
I don't see any validation.

Reply:
That is why I called it basic in the commit message. By now it does nothing, and is not even connected to a service. All i wanted is to display it on the feed page. Later on it will be improved.

Reply:
Got it. Other than that it looks good.